<launch>

    <param 
        name="/cam_node/cam_id" 
        type="int" 
        value="0" 
    />

    <node 
        pkg="face_processor" 
        type="cam_node.py" 
        name="cam_node" 
        output="screen" 
    />
    
</launch>